testng_unit_testing/src/test/java/testng/testRunner.java

```diff
@@ -0,0 +1,23 @@
+package testng;
+
+import org.testng.TestNG;
+import org.testng.xml.XmlSuite;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class testRunner {
+    public static void main(String[] args) {
```

Comment:
please add the ability to run the suits throw the Maven command. In this case, no need to have 'pubic static void main' entry point where you are not able to configure the specific suit e.g. smoke or regression

Reply:
For further modules